util/image-upload.js

```diff
@@ -1,9 +1,20 @@
 import { By, until } from "selenium-webdriver";
 import { defaultWait } from "../config.js";
 import path from "path";
+import { ExtendedBy } from "./selection.js";
 
-export async function uploadImage(driver, element_id, imagePath) {
-    const imageField = await driver.findElement(By.id(element_id));
+/**
+ * @param {any} driver 
+ * @param {string} element_id 
+ * @param {number} nthElement 
+ * @param {string} imagePath 
+ */
+export async function uploadImage(driver, element_id, nthElement, imagePath) {
+    let imageField;
+    if(nthElement === 0)
+        imageField = await driver.findElement(By.xpath(`(//*[starts-with(@id, '${element_id}')])`));
```

Comment:
I think we should try to stick to finding elements via `css` (or the tag name ^^)

Reply:
ergh yes. I was doing something else here before, will change this!